UT4MasterServer.Web/src/pages/Admin/Clients/hooks/use-client-options.hook.ts

```diff
@@ -0,0 +1,24 @@
+import { ITrustedGameServer } from "../../TrustedServers/types/trusted-game-server";
+import { IClient } from "../types/client";
+
+export function useClientOptions() {
+    const reservedNames = ['Launcher (our website)', 'Game', 'ServerInstance'];

```

Comment:
Names are not required to be set.
If there's a reason for you to check whether client is reserved or not, it should check the client id and not the name.